### PR TITLE
Fix 701

### LINKED ISSUE #701: Mitigate Clickjacking by Content-Security-Policy
**What's the issue?**
There is a almost new solution to avoid Clickjacking with `Content-Security-Policy` header.
I'll PR it ASAP. I'm not fluent in English. Can someone check grammar (also technically) please?

- [X] Assign me, please!

### PR DESCRIPTION
**Pull request by manindar-mohan**
This PR fixes #701.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Mitigate Clickjacking by Content-Security-Policy Using frame-ancestors directive
- Reference https://github.com/OWASP/wstg/issues/701
Thank you for your contribution!

### CODE REVIEW

**Comment by manindar-mohan** [comment]
Any issue I need to update? still, the pull request is pending.

**Comment by kingthorin** [comment]
It hasn’t been forgotten, just some life stuff in the way. I’ll tackle it next week when things should be less hectic.

File: document/4-Web_Application_Security_Testing/11-Client-side_Testing/09-Testing_for_Clickjacking.md [hunk 1]
```diff
@@ -262,6 +262,25 @@ Web proxies are known for adding and stripping headers. In the case in which a w
...
+The frame-ancestors directive was added to CSP Version 2. This means that browser support for frame-ancestors existed since 2015 in Chrome and Firefox, Safari 10+ or Edge 15+.
+
+The frame-ancestors CSP directive is not supported at all in Internet Explorer, would need to use the Edge browser instead.
+
+|Browser            | Lowest version  |
+|-------------------|-----------------|
+| Firefox (Gecko)   | 15              |
+| Opera             | 26              |
+| Safari            | 10              |
+| Chrome            | 40              |
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/11-Client-side_Testing/09-Testing_for_Clickjacking.md:282]: Is there really any benefit in including this information at this point. If it's been in available for 5+ years I'm not convinced this is worth including. If we "need" to say something I'd suggest more along the lines of:

> Supported in all mainstream modern browsers.

But I'm not sure that's is even of any benefit.

@ThunderSon @rbcsec thoughts? feedback?

Comment (kingthorin) [reply document/4-Web_Application_Security_Testing/11-Client-side_Testing/09-Testing_for_Clickjacking.md:282]: If this is kept "version" should get a capital as a column heading/title.

**Review by kingthorin — CHANGES_REQUESTED** [review]

File: document/4-Web_Application_Security_Testing/11-Client-side_Testing/09-Testing_for_Clickjacking.md (hunk 1 above)
Comment (manindar-mohan) [reply document/4-Web_Application_Security_Testing/11-Client-side_Testing/09-Testing_for_Clickjacking.md]: The heading is changed to capital, if the browser compatibility section is not required let me know will remove it. I was following the standard from the same document.

Comment (kingthorin) [reply document/4-Web_Application_Security_Testing/11-Client-side_Testing/09-Testing_for_Clickjacking.md]: Hmmm ya I see that it's also on the XFO entry. I guess that clean-up can be done separately.

**Pull request merged**